runtime/caml/osdeps.h

```diff
@@ -19,6 +19,8 @@
 #define CAML_OSDEPS_H
 
 #ifdef _WIN32
+#include <time.h>
```

Comment:
I wasn't keen on this addition (at least it's inside `_WIN32`) but it seemed more palatable than even more `#ifdef _WIN32` in sys.c